### ISSUE TITLE
index-patterns cannot be used when global tenant not enabled / .kibana index not existing

### ISSUE THREAD

**Issue by Schnitzel** [floragunncom/search-guard#566]
Searchguard Version: 6.4.2-23.1
Kibana Plugin Version: 6.4.2-15

I run into the same issue like https://groups.google.com/forum/#!msg/search-guard/WQtbweSyxow/v_ITQ7d9CwAJ

Basically I have 
```
searchguard.multitenancy.tenants.enable_global: false
searchguard.multitenancy.tenants.enable_private: false
```
configured in `config/kibana.yml`

Whenever I try to create an index-pattern I can do so, but I get the message:
![kibana_and_openshift_web_console](https://user-images.githubusercontent.com/750999/46587857-02017300-cade-11e8-98f6-f5fd722d2184.jpg)

These are the requests that are running on that page:
```
GET https://logs-db-ui-lagoon-develop.govcms.amazee.io/api/saved_objects/_find?type=index-pattern&fields=title&per_page=10000
```
![kibana_and_new_issue_ _floragunncom_search-guard](https://user-images.githubusercontent.com/750999/46587874-3ecd6a00-cade-11e8-86a7-c932593429a1.jpg)

```
POST https://logs-db-ui-lagoon-develop.govcms.amazee.io/api/saved_objects/_bulk_get
[{"id":"f3c23040-ca7e-11e8-9c79-1fb8278380ce","type":"index-pattern"}]
```
![kibana_and_new_issue_ _floragunncom_search-guard](https://user-images.githubusercontent.com/750999/46587882-602e5600-cade-11e8-8a02-7edecd757533.jpg)

Here you can see the error:
```
{"saved_objects":[{"id":"f3c23040-ca7e-11e8-9c79-1fb8278380ce","type":"index-pattern","error":{"statusCode":404,"message":"Not found"}}]}
```
Interestingly the status code is `200` though.

This results in Kibana not working at all. I can create new index-patterns, but the Discovery page is just WSOD.

To solve this I need to deploy kibana once with
```
searchguard.multitenancy.tenants.enable_global: true
searchguard.multitenancy.tenants.enable_private: false
```

Then just use the global tenant, in order that the `.kibana` index is created. And as soon as this is done, the tenants also work again.
I can even redeploy kibana with `enable_global: false` and it continues to work.
As soon as I remove the `.kibana` index via the elasticsearch API it starts to fail again. So it has to do with the existence of the `.kibana` index.

Another interesting thing:
The `.kibana` index just need to exist, no need to have anything in it, so a simple
```
curl -XPUT -k "http://localhost:9200/.kibana"
```
solves this as well.

**Comment by floragunncom** [comment]
Yes, this is a known bug. It's fixed and will be released with SG 23.2. As you wrote the issue shows if there is no .kibana index. Workaround is to let Kibana create it, and then switch the global tenant off.

**Comment by jochenkressin** [comment]
Fix is ready, tracked under ITT-1639, to be released with 23.2

**Comment by jochenkressin** [comment]
SG 23.2 has been released yesterday which fixes this issue.
